guide/docs/popular-topics/permissions.mdx

```diff
@@ -1,8 +1,382 @@
+
+2. If you can't view a channel, they you can't do anything in it.
+
+This rule automatically revokes "All Channel" permissions, as specified by <DocsLink reference="disnake.Permissions.all_channel" />,
+if the member doesn't have the "View Channel" permission.
+
+## Operating on permission values
+
+Since disnake v2.6, it is possible to do various operations on permission objects with minimal effort. While
+not something you'd usually use often, they can come in very handy when it comes to complex permission
+math you'd have to do manually otherwise.
+
+This section is meant as a companion to the <DocsLink reference="disnake.Permissions">official reference</DocsLink>,
+meaning that ideally you should read both.
+
+The following variables will be used throughout in the examples below.
+
+```python
+base = disnake.Permissions(add_reactions=True, manage_guild=True)
+same = disnake.Permissions(add_reactions=True, manage_guild=True)
+superset = disnake.Permissions(add_reactions=True, manage_guild=True, send_messages=True)
+subset = disnake.Permissions(add_reactions=True)
+different = disnake.Permissions(use_voice_activation=True)
+```
+
+### Basic
+
+These are operations that usually require no explanation. In any way, comments will guide you through.
+
+```python
+# Check that two permission object have same permissions set to the same value.
+assert base == same
+# Check that one (base) permission object has at least all of the granted permissions
+# of the second one granted too.
+assert base >= same
+# Check that one (base) permission object has all of the granted permissions
+# of the second one granted too, plus at least one extra.
+assert base > subset
+# Check that one (base) permission object has at maximum all of the granted permissions
+# of the second one granted.
+assert base <= same
+# Check that one (base) permission object has at maximum of the granted permissions
+# of the second one granted, minus one or more.
+assert base < superset
+# Check that one (base) permission object has at least one permission granted/revoked
+# that is revoked/granted in the second one.
+assert base != different
+```
+
+### Advanced
+
+If you're familar with bit operators in Python, then you'll feel right at home. Otherwise, read
+the comments to understand what does what.
+
+```python
+# Creates a new Permissions object with all granted permissions from both base and different.
+assert subset | different == disnake.Permissions(add_reactions=True, use_voice_activation=True)
+# Creates a new Permissions object with only permissions granted on both base and different.
+assert base & subset == disnake.Permissions(add_reactions=True)
+# Creates a new Permissions object with only permissions granted on one of base or different,
+# but not both.
+assert base ^ different
+
+# Notably, all of operators above support in-place updates using the `OP=` syntax, like so:
+# `base &= subset`.
```

Comment:
This part probably doesn't need to be mentioned here

Reply:
Did you mean it should be mentioned somewhere else or it doesn't need to be mentioned at all (in this case, why?)?

Reply:
My bad, I should've been more clear - I meant removing it entirely.
However, thinking about it again, keeping this line is also fine, so disregard my comment